alf/data_structures.py

```diff
@@ -118,7 +118,7 @@ def _create_timestep(observation, prev_action, reward, discount, env_id,
     create_tensor = lambda t: torch.as_tensor(t).detach()
     make_tensors = lambda struct: map_structure(create_tensor, struct)
     return TimeStep(
-        step_type=step_type.view(discount.shape),
+        step_type=torch.reshape(step_type, discount.shape),
```

Comment:
I believe @le-horizon had some reason for the original `view`. This function is only used in `metrics_test`

Reply:
I think it was because step_type is a scalar, and cannot be concatenated with other tensors in the TimeStep nest.  View is necessary to avoid complaints.